packages/fast-components-styles-msft/src/text-action/index.ts

```diff
@@ -0,0 +1,146 @@
+import { DesignSystem, withDesignSystemDefaults } from "../design-system";
+import {
+    ComponentStyles,
+    ComponentStyleSheet,
+    CSSRules,
+} from "@microsoft/fast-jss-manager";
+import { TextActionClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import { TextFieldClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import {
+    applyLocalizedProperty,
+    contrast,
+    Direction,
+    focusVisible,
+    toPx,
+} from "@microsoft/fast-jss-utilities";
+import {
+    applyMixedColor,
+    disabledContrast,
+    ensureNormalContrast,
+    ensuresBackgroundNormal,
+    foregroundNormal,
+    hoverContrast,
+    normalContrast,
+} from "../utilities/colors";
+import { density } from "../utilities/density";
+import { defaultHeight, maxHeight, minHeight } from "../utilities/height";
+import outlinePattern from "../patterns/outline";
+
+// Since MSFT button is already styled, we need to override in this way to alter button classes
+export const textFieldOverrides: ComponentStyles<
+    Partial<TextFieldClassNameContract>,
+    DesignSystem
+> = {
+    textField: {
+        height: "100%",
+        marginTop: "0",
```

Comment:
margin: "0"

Reply:
This can be removed all-together once the other PR is in